timelock-server/src/integTest/java/com/palantir/atlasdb/timelock/PaxosTimeLockServerIntegrationTest.java

```diff
@@ -374,7 +374,8 @@ public void fastForwardRespectsDistinctClients() {
 
         long currentTimestamp = timestampService.getFreshTimestamp();
         anotherClientTimestampManagementService.fastForwardTimestamp(currentTimestamp + ONE_MILLION);
-        assertEquals(currentTimestamp + 1, timestampService.getFreshTimestamp());
+        assertThat(timestampService.getFreshTimestamp())
+                .isBetween(currentTimestamp + 1, currentTimestamp + ONE_MILLION - 1);
```

Comment:
nit: we might as well write `.isStrictlyBetween(currentTimestamp, currentTimestamp + ONE_MILLION)` for this